### ISSUE TITLE
What do the different color and axis value mean. 

### ISSUE THREAD

**Issue by shivam-deepcompute** [philipperemy/keras-visualize-activations#5]
Hi @philipperemy,
I tried your tool, its very interesting to observe the activation-map. But I am facing one issue, i dont understand what the different colors mean. I have the below activation-map.

![screenshot from 2018-05-26 17-55-25](https://user-images.githubusercontent.com/33828942/40576080-5091cf40-610e-11e8-95f7-b38ac14e9813.png)
Is there some kind of reference I can refer to for different colors, generating in future.
Also i dont understand what values on x-axis and y-axis mean.
My test data contains - [[12345, 12345, 12345, 12345, 12345]]
Predicted Output - [[12345.001 12345.021 12345.    12344.999 12345.041]]

Any help is appreciated.

**Comment by philipperemy** [comment]
Sure. I use matplotlib to display the results. It's easy to understand for convolutional layers but less easy for model outputs.

Matplotlib uses the colormap called JET by default.

https://matplotlib.org/2.0.2/examples/color/colormaps_reference.html (check Misc colormaps around the bottom).

Blue color mean lowest values (~cold).
Red color mean highest values (~hot).
Green color mean average values (~mild).

In the README you can see the output of the softmax (Activation map of Softmax of LeNet. Yes it's a seven!). Everything is blue except one which is red. That means every value is very small except one. And that's what we expected. A very probability centered around 7 and everything close to 0.
